testfiles/AirflowNetwork_Simple_SmallOffice.idf

```diff
@@ -1467,7 +1464,10 @@
     For: AllOtherDays,       !- Field 13
     Until: 6:00,1.0,         !- Field 14
     Until: 13:00,0.0,        !- Field 16
-    Until: 24:00,1.0;        !- Field 18
+    Until: 24:00,1.0,        !- Field 18
+    Through: 12/31,          !- Field 1
+    For: AllDays,            !- Field 2
+    Until: 24:00,0.0;        !- Field 3
```

Comment:
This file shows diffs, but it is because this schedule was malformed and needed to be repaired.

Reply:
So, was the remainder of the year zeros before? Would that still be appropriate but with a warning? Just thinking out loud.

Reply:
I considered evaluating what it was doing before, but frankly I believe it is clearly telling the simulation to do two things at a point in the year, so I didn't worry about it.  I intend on just continuing to abort the simulation in this particular case.